buildkite/src/Pipeline/Filter.dhall

```diff
@@ -15,6 +15,7 @@ let tags: Filter -> List Tag.Type = \(filter: Filter) ->
     , TearDownOnly = [ Tag.Type.TearDown ]
     , ToolchainsOnly = [ Tag.Type.Toolchain ]
     , AllTests = [ Tag.Type.Lint, Tag.Type.Release, Tag.Type.Test ]
+    , Release = [ Tag.Type.Release ]
```

Comment:
I think you forgot to add Release tag in tag.dhall file

Reply:
There was already a `Tag.Type.Release` before my PR, you can see it was previously referenced in the line above this.